### DISCUSSION TITLE
Filters: Make label configurable from outside

### DISCUSSION THREAD

**Discussion by pehbehbeh** [naymspace/backpex#361] [Ideas]
Idea from @krns:

To increase the reusability, a label should be configurable from the outside. The following filter can only be used when the label is the same:

```elixir
defmodule EvuPlatformWeb.Filters.DateRange do
  @moduledoc false

  use Backpex.Filters.Range
  import EvuPlatformWeb.Gettext

  @impl Backpex.Filters.Range
  def type, do: :datetime

  @impl Backpex.Filter
  def label, do: dgettext("time_tracking", "Date")
end
```

**Comment by pehbehbeh** [comment]
This should be covered by #336 and #357.

**Comment by Flo0807** [comment]
Solved with https://github.com/naymspace/backpex/pull/226 @pehbehbeh
